metriq_gym/benchmarks/__init__.py

```diff
@@ -5,37 +5,37 @@
 from metriq_gym.benchmarks.clops import Clops, ClopsData
 from metriq_gym.benchmarks.quantum_volume import QuantumVolume, QuantumVolumeData
 from metriq_gym.benchmarks.bseq import BSEQ, BSEQData
-from metriq_gym.benchmarks.wormhole import Wormhole, WormholeData
```

Comment:
I am curious why this is removed. This is currently on the `main` branch, and we definitely don't want to remove this if it's going into `main` with your PR.